communication.md

```diff
@@ -1,120 +1,128 @@
+* Send/forward the email to an archive group in [Google Groups](software/googleapps.html). For maximum transparency, we recommend using these archive groups as a BCC recipient in all external email communication.
 
-* State of the business
-* Important events
-  * Changes to processes
-  * Important customers we got or lost
-  * Employee birthdays, anniversaries or other news
-* Recent achievements
+**I need to get immediate response on an urgent matter.**
 
-You can find the agenda for upcoming Town Halls on Asana and we encourage you to add your own agenda items that you find relevant to the company. We encourage everyone to also add relevant FYI items, e.g. plans for a longer vacation.
+* @-mention individual persons or address the whole group via `@here` on the appropriate [Slack](/software/slack.html) channel. Preface your post with "URGENT:".
 
 
-### Other meetings
+## Town Hall meeting
 
-Team discussions are expensive. They cost all participants time for preparation
-and attendance, and they keep them from doing other work. That's why we have
-these rules for team discussions:
+Once a week, we have a company-wide video call called "Town Hall". This call has the purpose of getting the whole team up-to-date with what's happening at freistil IT:
 
-1. Do we really need to talk? As an example, Dan McCarthy lists these
-   [legitimate goals](http://www.greatleadershipbydan.com/2013/11/how-to-make-team-meetings-less-painful.html):
-    * To solve problems, make decisions, or develop a plan
-    * To make important, timely announcements
-    * To share information that would be of interest to all team members
-    * Team (or maybe individual) recognition
-    * Learning
-1. Who really needs to take part in this discussion?
-1. Schedule a start, not an end – its over when its over, even if that's just 5
-   minutes.
-1. A team discussion invitation is only valid if it contains an agenda
-   describing the milestones and goals of the talk.
-1. Be on time!
-1. No multi-tasking. No device usage unless it's really necessary.
-1. Scheduled discussions are not for information sharing — that should be done
-   before via email and/or agenda.
-1. If you're not getting anything out of the discussion, leave.
-1. Agree to action items, if there are any, at the conclusion of the discussion.
-1. Don't feel bad about calling people out on any of the above; it's the right
-   thing to do!
+* Recent business achievements and failures
+* Important events
+  * Changes to processes
+  * Employee birthdays, anniversaries and other news
+
+Town Hall is also a great opportunity to ask questions and announce things that are relevant for the whole company. You can find the agenda for upcoming Town Halls on Asana and we encourage you to add your own agenda items.
```

Comment:
A direct link to the Town hall agenda would make it easier to jump there: http://f7it.com/f7townhall